ignite/services/scaffolder/module.go

```diff
@@ -56,6 +56,7 @@ var (
 		"account":                    {},
 		"block":                      {},
 		"broadcast":                  {},
+		"consumer":                   {}, // ICS consumer module
```

Comment:
Since the consumer module doesn't have the app wiring, we should inject the app module into the client side to avoid crashes when we export the genesis. It can be added here:

https://github.com/ignite/cli/blob/9025f3aa1f83ccd38d187f0b5f89ad15dc344bef/ignite/templates/app/files-consumer/app/ibc.go.plush#L46-L59

Reply:
It's there: https://github.com/ignite/cli/blob/9025f3aa1f83ccd38d187f0b5f89ad15dc344bef/ignite/templates/app/files-consumer/app/ibc.go.plush#L53 on your snippet
And there for the client side: https://github.com/ignite/cli/blob/9025f3aa1f83ccd38d187f0b5f89ad15dc344bef/ignite/templates/app/files-consumer/app/ibc.go.plush#L233

Genesis exports do work.